kubernetes/deploy.sh

```diff
@@ -2,8 +2,6 @@
 
 # Deploys CoreDNS to a cluster currently running Kube-DNS.
 
-set -eo pipefail
```

Comment:
This change crept in https://github.com/coredns/deployment/pull/99.
Do we need to remove it in the rollback script as well?

Reply:
yes - it thwarts the error checking there.

Reply:
#145